app/frontend/src/features/messages/Messages.stories.tsx

```diff
@@ -21,3 +44,39 @@ const Template: Story<any> = (args) => <Messages {...args} />;
 
 export const Assembled = Template.bind({});
 Assembled.args = {};
+
+const message: MessageProps["message"] = {
+  messageId: 1,
+  authorUserId: 2,
+  // time?: google_protobuf_timestamp_pb.Timestamp.AsObject,
+  text: { text: "testtext" },
+  time: { seconds: Math.floor(+new Date(2020, 0, 1) / 1e3), nanos: 0 },
+  // chatCreated?: MessageContentChatCreated.AsObject,
+  // chatEdited?: MessageContentChatEdited.AsObject,
+  // userInvited?: MessageContentUserInvited.AsObject,
+  // userLeft?: MessageContentUserLeft.AsObject,
+  // userMadeAdmin?: MessageContentUserMadeAdmin.AsObject,
+  // userRemovedAdmin?: MessageContentUserRemovedAdmin.AsObject,
+  // hostRequestStatusChanged?: MessageContentHostRequestStatusChanged.AsObject,
```

Comment:
I'm guessing these are properties in the response but not needed for what you need to render here for Storybook?

Reply:
Indeed, copy pasted from the definition. I left them here to easily add more properties.